delta-api/src/main/java/io/cdap/delta/api/assessment/TableAssessorSupplier.java

```diff
@@ -33,4 +36,17 @@
    * @throws Exception if the table assessor could not be created
    */
   TableAssessor<T> createTableAssessor(Configurer configurer) throws Exception;
+
+  /**
+   * Create a table assessor that will check if there will be potential problems replicating tables.
+   *
+   * @param configurer configurer used to instantiate plugins
+   * @param tables tables to be replicated
+   * @return a table assessor
+   * @throws Exception if the table assessor could not be created
+   */
+  default TableAssessor<T> createTableAssessor(Configurer configurer, List<SourceTable> tables) throws Exception {
```

Comment:
This method returns `TableAssessor` which is meant for the single table only, right? How will the list of tables be used by `TableAssessor`? Can you provide some sample pseudo code for this API for my understanding?

Reply:
This is not for a single table , the TableAssessor has two interfaces :
 TableAssessment assess(T tableDescriptor).  -- assess a single table
Assessment assess() -- assess the whole replicator pipeline